pkg/services/sqlstore/alert_notification.go

```diff
@@ -179,8 +197,19 @@ func UpdateAlertNotification(cmd *m.UpdateAlertNotificationCommand) error {
 		current.Name = cmd.Name
 		current.Type = cmd.Type
 		current.IsDefault = cmd.IsDefault
+		current.NotifyOnce = cmd.NotifyOnce
 
-		sess.UseBool("is_default")
+		if cmd.Frequency == "" {
+			return fmt.Errorf("Alert notification frequency required")
```

Comment:
This error should be created in https://github.com/grafana/grafana/blob/master/pkg/models/alert_notifications.go#L8 and used from here. That way the customer of this command can act differently depending on type of error. Like this https://github.com/grafana/grafana/blob/master/pkg/models/dashboards.go#L15